From 7553947c912ca095f1f4baa7ff924ccbee62feb9 Mon Sep 17 00:00:00 2001
From: Pete Beardmore <pete.beardmore@msn.com>
Date: Tue, 12 Sep 2017 16:59:03 +0100
Subject: harmonise the widgetbars initial pane sort order

-with no sort order set, make it the same as the order without widgetbars
---
 quodlibet/quodlibet/qltk/quodlibetwindow.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/quodlibet/quodlibet/qltk/quodlibetwindow.py b/quodlibet/quodlibet/qltk/quodlibetwindow.py
index 2d16671..f2d003d 100644
--- a/quodlibet/quodlibet/qltk/quodlibetwindow.py
+++ b/quodlibet/quodlibet/qltk/quodlibetwindow.py
@@ -916,8 +916,10 @@ class QuodLibetWindow(Window, PersistentWindowMixin, AppWindow):
         order = config.getstringlist('memory', 'pane_order') # default only
 
         widgetbars = self.widgetbars_enabled
-        if widgetbars:
-            order = config.getstringlist('plugins', 'widgetbars_pane_order')
+        widgetbars_order = \
+            config.getstringlist('plugins', 'widgetbars_pane_order')
+        if widgetbars and widgetbars_order:
+            order = widgetbars_order
 
         if 'main' not in order:
             order = ['main'] + order
-- 
1.8.4

